be/src/common/config.h

```diff
@@ -826,4 +826,12 @@ CONF_String(directory_of_inject,
 CONF_Int64(max_length_for_to_base64, "200000");
 // Used by bitmap functions
 CONF_Int64(max_length_for_bitmap_function, "1000000");
+
+CONF_Bool(block_cache_enable, "false");
```

Comment:
yes, maybe we use directory name like `block_cache`. anyway it's much better and specific than `cache`.